lib/WTSI/NPG/iRODS/PublisherFactory.pm

```diff
@@ -1,12 +1,41 @@
 package WTSI::NPG::iRODS::PublisherFactory;
 
-use namespace::autoclean;
-use Moose;
+use strict;
+use warnings;
+use Moose::Role;
```

Comment:
Why changed to a role? These methods will now appear in the public API of anything that consumes them - do they need to?